### ISSUE TITLE
Treex view

### ISSUE THREAD

**Issue by natalink** [ufal/lindat-kontext#9]
Adding functionality to visualize syntactically annotated sentences as trees: 89c6c27, 454fcf1
1. The visualisation tool is taken from Treex-Web: https://github.com/ufal/js-treex-view/tree/releases cloned into /opt/lindat/kontext/js-treex-view && cp js-treex-view.js ../public/files/js
2. js-treex-view.js processes json and visualizes a respective tree. The process of generating json files relies on the Treex block [Treex::Block::Write::ViewJSON](https://github.com/ufal/treex/blob/master/lib/Treex/Block/Write/ViewJSON.pm) . It is necessary to install [treex](https://github.com/ufal/treex/) in
   order to do the conversion to JSON via this block.
3. Data:
   - JSON files from which the trees are generated are stored in the respective folder (corpname) in /opt/lindat/kontext-data/corpora/view-treex , the filename is an id of a sentence, e.g. s1.json, s2.json ...
   - vertical file:

```
<doc id="xyz">
<s id="s1">
token    lemma    POS-tag    afun ...
token    lemma    POS-tag    afun ...
token    lemma    POS-tag    afun ...
...
</s>
<s id="s2">
token    lemma    POS-tag    afun ...
token    lemma    POS-tag    afun ...
token    lemma    POS-tag    afun ...
...
</s>
...
</doc>
```

 Access to the json files are provided by method _view_tree_ in actions.py 
In analogy with the speech files, the path is specified in the config.xml:

```
 <view_treex_files_path>/opt/lindat/kontext-data/corpora/view_treex</view_treex_files_path>
```

**Comment by natalink** [comment]
89c6c27

**Comment by natalink** [comment]
Unfortunately, the point to which corpus will have treex view option is implemented in the core code of KonText 0.5: 
https://github.com/ufal/lindat-kontext/blob/454fcf1e3880bf22b3f676a0747c481febcd1e53/templates/view.tmpl#L196
and
https://github.com/ufal/lindat-kontext/blob/454fcf1e3880bf22b3f676a0747c481febcd1e53/lib/conclib.py#L460
In the new version of KonText, this needs to be changed (included as an option to corpus into config.xml?? )

**Comment by vidiecan** [comment]
see #53
